app/assets/stylesheets/components/_fieldset.scss

```diff
@@ -0,0 +1,30 @@
+@import "helpers/variables";
+@import "helpers/clearfix";
+
+.gem-c-fieldset {
+  margin: 0 0 $gem-spacing-scale-4;
+
+  @include media(tablet) {
+    margin-bottom: $gem-spacing-scale-5;
+  }
+
+  padding: 0;
+  border: 0;
+  @include gem-h-clearfix;
+}
+
+.gem-c-fieldset__legend {
+  // Fix legend text wrapping in Edge and IE
+  // 1. IE9-11 & Edge 12-13
+  // 2. IE8-11
+  box-sizing: border-box; // 1
```

Comment:
I'm not aware of a mixin for box-sizing

Reply:
https://github.com/alphagov/govuk_frontend_toolkit/blob/452f82e7538d7f5b8f73ee7d81abea9bc857af13/stylesheets/_css3.scss#L68

Reply:
I don't think it's useful to use this mixin since the support for the unprefixed version is so high at 98%

https://caniuse.com/#feat=css3-boxsizing